fernet.py

```diff
@@ -38,12 +38,12 @@ def __init__(self, key):
     def generate_key(cls):
         return base64.urlsafe_b64encode(os.urandom(32))
 
-    def encrypt(self, data) -> bytes:
```

Comment:
I am kind of enthusiastic about type hints, any reason, why they should go away?